aws-lambda/go/README.md

```diff
@@ -0,0 +1,52 @@
+# Steps to test Sentry SDK integration with Node Lambda function:
+
+[Demo Video](https://www.loom.com/share/e3625614df2f44f3bb96286ad7212f99)
+
+## Create a development package
+1. Clone the git repo on your development machine.
+
+2. Build your executable by running
+
+```
+GOOS=linux go build main.go
+```
+
+3. Create a zip
+```
+zip function.zip main
+
+// Include source code in zip to view source code in stacktrace.
+// Or, setup your Sentry<>Github integration instead.
```

Comment:
Is there a good documentation link to add here? Getting the integration to work will probably need a way to match the version of the code by commit hash out something else which is not part of the example.

Reply:
are you talking about getting suspect commits to work? to just get the source code to show, it seemed pretty easy. I'm pretty sure I just have the Sentry<>Github integration setup. I can put a link for that. Beyond that, I'm not going to add any other steps here. They can contact us if they need additional help on that.

Reply:
_Sorry I was unavailable last days to look into this._

Thanks for adding the link. Pointer to docs is the way to go here, instead of replicating all steps in this comment. I meant
https://docs.sentry.io/product/integrations/github/#stack-trace-linking

> Stack trace linking takes you from a file in your Sentry stack trace to that same file in your source code. If you have commit tracking set up in Sentry, we can take you to the exact version (using the commit associated with the event) of the source code when the error occurred. **Otherwise we'll link you to the current state of the source code (using the default branch)**.

So the integration takes you to the latest commit, which may or may not match what was deployed to AWS Lambda. For a quick demo it works, but to make it work correctly as code is being changed and multiple versions being deployed, there's more config needed.